portfolio/src/main/webapp/script.js

```diff
@@ -32,3 +32,174 @@ function randomizeImage() {
   imageContainer.innerHTML = '';
   imageContainer.appendChild(imgElement);
 }
+
+/**
+ * Handles response by converting it to text and passing the result to
+ * addNameToDom().
+ */
+function handleResponse(response) {
+  console.log('Handling the response.');
+
+  // response.text() returns a Promise, because the response is a stream of
+  // content and not a simple variable.
+  const textPromise = response.text();
+
+  // When the response is converted to text, pass the result into the
+  // addNameToDom() function.
+  textPromise.then(addNameToDom);
+}
+
+/** Adds a name to the DOM. */
+function addNameToDom(name) {
+  console.log('Adding name to dom: ' + name);
+
+  const NameContainer = document.getElementById('name-container');
+  NameContainer.innerText = name;
+}
+
+/**
+ * Fetches comments from the servers and adds them to the DOM.
+ */
+async function getComments() {
+  await fetch('/data').then(response => response.json()).then((comments) => {
+    const comment_idListElement = document.getElementById('comment-list');
+    comments.forEach((comment_id) => {
+      comment_idListElement.appendChild(createListElement(comment_id));
+    })
+  });
+}
+
+/** Creates an <li> element containing text. */
+function createListElement(comment_id) {
+  const commentElement = document.createElement('li');
+  commentElement.className = 'comment_id';
+
+  const comment_textElement = document.createElement('span');
+  comment_textElement.innerText = comment_id.comment_text;
+
+  commentElement.appendChild(comment_textElement);
+  return commentElement;
+}
+
+async function deleteComments() {
+  await fetch('/delete-data', { method: 'POST'});
```

Comment:
Here await is useful as we do not want to reload the page unless delete is finished.